src/runtime/cuda.cpp

```diff
@@ -153,16 +166,15 @@ WEAK int halide_default_cuda_acquire_context(void *user_context, CUcontext *ctx,
     if (local_val == nullptr) {
         if (!create) {
             *ctx = nullptr;
-            return 0;
+            return halide_error_code_success;
         }
 
         {
             ScopedMutexLock spinlock(&context_lock);
             local_val = context;
             if (local_val == nullptr) {
-                CUresult error = create_cuda_context(user_context, &local_val);
-                if (error != CUDA_SUCCESS) {
-                    return error;
+                if (auto result = create_cuda_context(user_context, &local_val); result != halide_error_code_success) {
```

Comment:
stylistically this looks awkward to me. What do you think about a line break after the semicolon when we do this?

Reply:
I thought the line-break version also looked awkward, but let's switch to that and see what you think.